### DISCUSSION TITLE
Archiving repositories

### DISCUSSION THREAD

**Discussion by Ein-Tim** [corona-warn-app/cwa-website#3503] [Q&A] [answered]
In https://www.coronawarn.app/en/faq/results/#ramp_down_reports it says: "[...] issues can also no longer be opened on GitHub as of May 16, 2023.".

It is May 16, 2023 now and I am still able to open issues in nearly all repositories. 
I'm opening this discussion to get public feedback on what the plan on this is.

**Comment by MikeMcC399** [comment]
@Ein-Tim 

I was also wondering about this. If a repository is archived, then no more changes can be made. I don't believe that there is a setting on GitHub which prevents new issues from being opened, whilst at the same time allowing open issues to be visible so they can be worked on and closed.

From a practical point of view this is probably not so much of a problem as by now there are almost no new issues coming in, and even some of the issues that were reported after May 1, 2023 have not been responded to.

I would expect that all [corona-warn-app](https://github.com/corona-warn-app) repositories, except this one ([cwa-website](https://github.com/corona-warn-app/cwa-website)), to be archived around the final EOD data May 31, 2023. Since it has been stated that the website will continue for science blogs, then the GitHub repo for the website would need to be kept open, unless the website is going to be sourced from a different repo.

GitHub help [Archiving repositories](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories) says: 
> We recommend that you close all issues and pull requests, as well as update the README file and description, before you archive a repository.

**Comment by MikeMcC399** [comment, marked as answer]
35 repositories are now archived (leaving only 4 not archived: this repo, [Android](https://github.com/corona-warn-app/cwa-app-android) & [iOS](https://github.com/corona-warn-app/cwa-app-ios) app and [.github](https://github.com/corona-warn-app/.github)):

https://github.com/corona-warn-app?q=&type=archived

It's slowly getting time to say goodbye!

**Reply by Ein-Tim** [reply]
This repo will stay unarchived and continued to be used.

**Reply by MikeMcC399** [reply]
@Ein-Tim 

> This repo will stay unarchived and continued to be used.

Thank you for the confirmation. Perhaps the [Repositories](https://github.com/corona-warn-app/cwa-website#repositories) section in the README should be amended to say that the list of other repositories is now archived and no longer maintained?

**Reply by Ein-Tim** [reply]
I will take care of that!

**Comment by MikeMcC399** [comment]
@Ein-Tim 

The text in [What if the app has malfunctions/has errors, will they not be fixed? Can I still report those bugs?](https://www.coronawarn.app/en/faq/results/#ramp_down_reports) which you quoted is not entirely correct. It says:

> Until April 30, 2023, error reports can still be transmitted to the development team. After this date, any bugs and errors can be reported until May 31, 2023. However, bug fixes can only be produced for errors received up to May 15, 2023. Accordingly, issues can also no longer be opened on GitHub as of May 16, 2023.

It would need to say "Accordingly, issues regarding the Android and iOS apps which are opened in GitHub after May 15, 2023 can no longer be addressed." It is still possible to open issues and there is no way (that I know of) to prevent this. There is a text addition lined up in the app repos "Löschen Sie die App nicht; wenn diese reaktiviert oder anderweitig weiterentwickelt wird, werden Sie informiert." and to process this the repos cannot be archived.

I don't think that the additional sentence about not deleting the app makes sense if the app is no longer in the app stores. This seems to belong to the political message that the app has been put into [hibernation](https://dictionary.cambridge.org/dictionary/english/hibernation) rather than [put to sleep](https://dictionary.cambridge.org/dictionary/english/put-to-sleep).

**Reply by Ein-Tim** [reply]
I will try to get information on this but I assume that the iOS & Android repo will be archived also.

**Reply by MikeMcC399** [reply]
@Ein-Tim 

> I will try to get information on this but I assume that the iOS & Android repo will be archived also.

I assume this also and that the archiving will take place around May 31, 2023.

**Reply by Ein-Tim** [reply]
iOS & Android will be archived as soon as it is certain that no more hot fixes are required.

**Reply by MikeMcC399** [reply]
> iOS & Android will be archived as soon as it is certain that no more hot fixes are required.

That makes sense.

**Reply by MikeMcC399** [reply]
iOS & Android are now also archived.
